.github/workflows/ts-rpc-test.yml

```diff
@@ -33,12 +33,12 @@ jobs:
 
       - name: Run Create Channels script
         # TODO: We could write a test specific script that creates channels and checks the results
-        run: npx ts-node ./scripts/client-runner.ts create-channels -w 300000 &> output.log
```

Comment:
Does this get rid of the duplicate log files that are archived by the CI?

Reply:
Yes we now upload a single log file for the servers, and print the client's output straight to the console.